packages/ios-client/Sources/FishjamClient/media/Tracks/VideoTrack.swift

```diff
@@ -5,7 +5,7 @@ open class VideoTrack: Track {
         mediaTrack: RTCVideoTrack, endpointId: String, rtcEngineId: String?, metadata: Metadata = Metadata(),
         id: String = UUID().uuidString
     ) {
-        super.init(mediaTrack: mediaTrack, endpointId: endpointId, rtcEngineId: rtcEngineId, metadata: metadata)
+        super.init(mediaTrack: mediaTrack, endpointId: endpointId, rtcEngineId: rtcEngineId, metadata: metadata, id: id)
```

Comment:
Would it make sense to do not have `id` as optional param in `Track`?

Reply:
I think it would make sense for id to be a required param in initializer and on the other hand it would make sense to be optional param. The reason I made it optional is android shares the same behaviour.